City-of-Helsinki/helsinki-paatokset
# PP-448: Decisionmaker page fixes

**To test**
- Checkout branch
- Run `make drush-cim drush-cr`
- Login to the container with `make shell` and run `drush mim ahjo_decisionmakers:all && drush mim ahjo_decisionmakers:all_sv && drush mim ahjo_trustees:council && drush mim ahjo_meetings:latest`
- Import some TPR Units with: `drush migrate:import tpr_unit --limit=200`
- Open the City council page: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto
  - Scroll down to the members list. The "Show deputies" checkbox should be visible 
  - Go to the UI translation page: https://helsinki-paatokset.docker.so/fi/admin/config/regional/translate
  - Translate "Deputy councillor" as "Varajäsen" and "Ersättare"
  - Reload the city council page. The labels should be translated and the "Show deputies" checkbox should still be visible
  - Test that the checkbox actually works (no one with the "Varajäsen" title should be visible by default, only after the checkbox is active)
- Edit the city council page and add some references in the "Contact information" section (these are the TPR units)
  - Doesn't matter which ones you add
  - After saving, check that they are visible underneath the sidebar navigation
  - Test the page on mobile resolutions. The sidebar navigation should be replaced by a dropdown, and the contacts should be moved to the bottom of the page
